.github/workflows/codeql.yml

```
@@ -42,12 +42,12 @@ jobs:
     - name: Checkout submodules
       run: git submodule update --init --recursive
-    
-    # Try to force .NET 6 build, this doesn't work atm well because we have projects that uses .net2.0 standard
-    #- name: Setup .NET
-    #  uses: actions/setup-dotnet@v1
-    #  with:
-    #    dotnet-version: '6.0'
+
+    # Use the same .NET setup instruction used at dotnet.yml
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
```

actionlint (1.7.9)

the runner of "actions/setup-dotnet@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)